src/main.jl

```diff
@@ -1,19 +1,19 @@
-const default_registries = Pkg.RegistrySpec[Pkg.RegistrySpec(name = "General",
-                                                             uuid = "23338594-aafe-5451-b93e-139f81909106",
-                                                             url = "https://github.com/JuliaRegistries/General.git")]
+const DEFAULT_REGISTRIES = Pkg.RegistrySpec[Pkg.RegistrySpec(name = "General", uuid = "23338594-aafe-5451-b93e-139f81909106", url = "https://github.com/JuliaRegistries/General.git")]
```

Comment:
I'm just not buying that this is a style improvement. But I have no involvement in this code so consider it a drive-by comment.